nodes/bug_master

```diff
@@ -52,23 +52,59 @@ def clbk_laser(msg):
 def clbk_joy(msg):
     print(msg.header.seq)
     #output structure header, axes, buttons
-    
-    # TODO Hardcoded for now
-    step = 1000  # unit
-
-    mx = msg.axes[3] # 4th array item
-    my = msg.axes[4] # 5th array item
-    print("Joy x:"+ str(mx), " y:" + str(my))
-
-    if(my >= 0.8):
-        #Move 
-        print("Requested Move...")
-        state_variables_.goal_pos.x = step # state_variables_.position.x + step
-
-    if(my <= -0.8):
-        #stop
-        print("Requested Stop!")
-        state_variables_.goal_pos = state_variables_.position
+    joystick = rospy.get_param('joyType')
```

Comment:
Wrap the "input" mechanism into a class, and create a child class called "joystick". If we have different input mechanisms (ex: voice, keyboard, etc.), we'll need to refactor this code.

Reply:
Yes, discussed and moving to next milestone after we make sure all individual pieces are working fine.